src/style/animatableManager.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
+
+
+/** Definitions */
+const definitions = {
+  basicListEntrance: {
+    from: {opacity: 0, translateY: 20},
+    to: {opacity: 1, translateY: 0},
+  }, 
+  cardEntrance: {
+    from: {opacity: 0, translateY: 40},
+    to: {opacity: 1, translateY: 0},
+  },
+};
+
+class AnimatableManager {
+  loadCustomDefinitions(costumDefinitions) {
```

Comment:
The name is wrong, this method doesn't load but get definitions. 
Having said that, I actually do think should it should load the definitions, but should load only the custom definitions, while our definitions should be loaded in the manager constructor. 

I think it should work similar to our other presets. Where developers get uilib's definitions out of the box but still can load more if they want.

Reply:
Done.